typedb/concept/thing/entity.py

```diff
@@ -19,45 +19,15 @@
 # under the License.
 #
 
-import typedb_protocol.common.concept_pb2 as concept_proto
+from __future__ import annotations
 
-from typedb.api.concept.thing.entity import Entity, RemoteEntity
-from typedb.api.concept.type.entity_type import EntityType
-from typedb.concept.proto import concept_proto_reader
-from typedb.concept.thing.thing import _Thing, _RemoteThing
+from typedb.api.concept.thing.entity import Entity
+from typedb.concept.thing.thing import _Thing
+from typedb.concept.type import entity_type
+from typedb.typedb_client_python import entity_get_type
 
 
 class _Entity(Entity, _Thing):
 
-    def __init__(self, iid: str, is_inferred: bool, entity_type: EntityType):
-        super(_Entity, self).__init__(iid, is_inferred)
-        self._type = entity_type
-
-    @staticmethod
-    def of(thing_proto: concept_proto.Thing):
-        return _Entity(concept_proto_reader.iid(thing_proto.iid), thing_proto.inferred, concept_proto_reader.type_(thing_proto.type))
-
-    def get_type(self) -> "EntityType":
-        return self._type
-
-    def as_remote(self, transaction):
-        return _RemoteEntity(transaction, self._iid, self.is_inferred(), self.get_type())
-
-    def as_entity(self) -> "Entity":
-        return self
```

Comment:
`as_foo()` is not overridden on many of the concepts as far as I can see. Won't they just raise an exception based on the `Concept` implementation?

Reply:
Yeah, we should put them back.